website/src/components/ask-ai/AISearch.jsx

```diff
@@ -78,7 +78,7 @@ const AISearch = forwardRef((props, ref) => {
         registerAISearch(query);
 
         try {
-            const apiUrl = "https://hook.eu1.make.com/zw41brw94vfiwuue91riqmoe2ms2nhf6";
+            const apiUrl = "https://ai.appsmith.com/api/v1/docs/query";
             const response = await fetch(apiUrl, {
                 method: 'POST',
```

Comment:
The response time is still a bit slow only around 25-30 seconds.

Reply:
The response time is better that what we have now. Over time this will get faster as the llm gets better. 
In any case we can work on them without any changes to the url. So I would suggest merging this PR.

Reply:
Once you confirm the above findings, we can go ahead and merge the PR.